flatcar-release-mirror.sh

```diff
@@ -124,13 +125,17 @@ download_folder() {
         if [[ -z "$above_version" ]] || [[ -z "$is_version" ]] || [ "$version" -gt "$above_version" ]; then
           mkdir -p "$link"
           cd "$link"
-          download_folder "${url}$link" || return 1
+          download_folder "${url}$link" "$arch" || return 1
           cd ..
         else
           echo "Skipping folder version: $link" >> "$logfile"
         fi
       else
-        download_file "${url}$link" "$link" || return 1
+        for ar in $arch; do
```

Comment:
I think you can save time by moving this check around `download_folder` and check `[[ "${url}$link" == *"${ar}-usr"* ]]`. Also note that you should quote the first part.

Reply:
Ah, right, it would only work if we reverse the check to skip if the URL contains an unwanted arch.

If you don't want to rework this and thinks are ok for you runtime-wise I'm fine leaving it as is.

Reply:
I think rework takes a lot effort to the script, not worth it for saving the little time.